src/tactic/core.lean

```diff
@@ -255,6 +256,21 @@ do e ← tactic.get_env,
              then s.insert d.to_name d else s),
    pure xs
 
+/-- `get_decls_from` returns a dictionary mapping names to their corresponding declarations.
```

Comment:
What is the expected form of `fs`? Is it listing imports (`data.real.basic`)? Full file paths? Relative file paths? Only the file name?